FhirToDataLake/native/parquet/csharp/src/ParquetConverter.cs

```diff
@@ -46,12 +46,18 @@ public Stream ConvertJsonToParquet(string schemaType, string inputJson)
             // Output buffer pointer
             IntPtr outputPointer = IntPtr.Zero;
 
+            if (inputJson == null)
+            {
+                throw new ParquetException(ParquetConverterErrorCodes.ReadInputJsonError, "Input Json is null.");
```

Comment:
use the same string variable as `ParquetException` to keep consistent for the error message.

Reply:
Pre offline discussion, keep current string and move it into rexs.